src/terrain/globe_raster_program.js

```diff
@@ -78,15 +94,31 @@ const globeRasterUniformValues = (
     globeMercatorMatrix: Float32Array,
     zoomTransition: number,
     mercCenter: [number, number],
-    gridMatrix: ?Array<number>
+    gridMatrix: ?Array<number>,
+    isAntialias: boolean,
+    frustumDirTl: [number, number, number],
+    frustumDirTr: [number, number, number],
+    frustumDirBr: [number, number, number],
+    frustumDirBl: [number, number, number],
+    globePosition: [number, number, number],
+    globeRadius: number,
+    viewport: [number, number]
 ): UniformValues<GlobeRasterUniformsType> => ({
     'u_proj_matrix': Float32Array.from(projMatrix),
     'u_globe_matrix': globeMatrix,
     'u_merc_matrix': globeMercatorMatrix,
     'u_zoom_transition': zoomTransition,
     'u_merc_center': mercCenter,
     'u_image0': 0,
-    'u_grid_matrix': gridMatrix ? Float32Array.from(gridMatrix) : new Float32Array(9)
+    'u_grid_matrix': gridMatrix ? Float32Array.from(gridMatrix) : new Float32Array(9),
+    'u_is_antialias': isAntialias,
```

Comment:
`u_is_antialias` is a uniform1i, which means that's it's uploaded to the gpu as by allocating an integer. To translate the boolean to integer representation safely you can write:
```suggestion
    'u_is_antialias': +isAntialias,
```

Reply:
ah thank you!